pipelines/data_embedding_pipeline/data_embedding_pipeline.py

```diff
@@ -33,3 +33,24 @@ def data_embedding_pipeline() -> None:
         chunk_size=2000,
         chunk_overlap=50,
     )
+    embed_data(
```

Comment:
There are 3 `embed_data` step here. The merge from `develop` must have brought this. Be careful about the values specified for `chunk_size` and `chunk_overlap`.

Reply:
Ahhh, missed that, which chunk size is the correct one for nhs?